mono/metadata/icall.c

```diff
@@ -5888,6 +5892,28 @@ ves_icall_System_Environment_get_NewLine (void)
 #endif
 }
 
+ICALL_EXPORT MonoBoolean
+ves_icall_System_Environment_GetIs64BitOperatingSystem (void)
+{
+#if SIZEOF_VOID_P == 8
+	return TRUE;
+#else
+#ifdef HOST_WIN32
+	MonoBoolean isWow64Process = FALSE;
+	if (IsWow64Process (GetCurrentProcess (), &isWow64Process) {
```

Comment:
I replaced it with `gboolean` to be consistent with the other usages in this file. I tried it on my windows box and it works.